bin/spack-install-helper-10-preprocessing

```diff
@@ -8,6 +8,22 @@
 # A100 = 8.0
 # H100 = 9.0
 
+SPACK_REPO_LIST=()
+
+while true
+do
+    case $1 in
+        --add-repo )
+            SPACK_REPO_LIST+=($2)
```

Comment:
probably never happening, but if someone ever comes up with weir whitespace directories, we would nee `"$2"`, i.e. quote the argument